status/reporters.go

```diff
@@ -674,6 +674,11 @@ func (r *GitHubReporter) ReportStatusForSnapshot(k8sClient client.Client, ctx co
 		return err
 	}
 
+	if len(statuses.GetStatuses()) == 0 {
```

Comment:
We may also log some message for this type scenario, like
```
logger.Info("No test result to report to GitHub, skipping",
"snapshot.Namespace", snapshot.Namespace, "snapshot.Name", snapshot.Name)
```

Reply:
added